Microsoft.DotNet.Interactive/StreamKernelEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.DotNet.Interactive
+{
+    public class StreamKernelEvent
+    {
+        public int Id { get; set; }
+        public string Type { get; set; }
```

Comment:
For consistency this should be `EventType` or else `StreamKernelCommand.CommandType` should be renamed to `Type`.